wordpress-mobile/WordPress-iOS
# Convert the theme service to use WordPressComRestAPI.

Refs #5245 

To test:
 - Login in the app using a wp.com account
 - Go to the theme section, and test the following:
  - List the themes
  - Activate a theme
  - Search theme

Check if all is working has before.

Needs review: @diegoreymendez
